node-fs-readfile/cat.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const length = process.argv.length;
+const count = 2;
+
+function getAnytext(count) {
+  if (count < length) {
+    fs.readFile(process.argv[count], 'utf-8', (err, data) => {
+      if (err) throw err;
+      console.log(data);
+    });
+    count++;
+    getAnytext(count);
```

Comment:
this `getAnytext(count)` call needs to be moved into the `readFile` callback to guarantee that it runs after the previous read completes